library_panel_test.go

```diff
@@ -161,7 +196,20 @@ func TestLibraryPanelGet(t *testing.T) {
 		t.Errorf("Invalid uid - %s, Expected %s", resp.UID, "V--OrYHnz")
 	}
 
-	resp, err = client.LibraryPanelByUID("V--OrYHnz")
+	for _, code := range []int{401, 403, 404} {
+		server.code = code
```

Comment:
Can you get rid of this assignment? It isn't used in the code after this.

Reply:
i honestly don't fully understand this portion; i copied it from elsewhere :)

Reply:
9bffaa40d14f7252c78c238842d38b6a23c0def9

Reply:
op, looks like it was definitely needed: https://drone.grafana.net/grafana/grafana-api-golang-client/231/1/2

Reply:
2d400ae9a00d7ae3e54300cf30f164cbbd4db6de

Reply:
This is really odd. I'll try and look into it later.